### PR TITLE
[RFC][UN-10462] Add calendar icon to data input

### PR DESCRIPTION
**Pull request by henrmota**
https://uniplaces.atlassian.net/browse/UN-10462

screen shot 2017-09-25 at 11 57 50

### CODE REVIEW

File: addon/styles/partials/_uni-datepicker-input.scss [hunk 1]
```diff
@@ -7,4 +7,32 @@
     margin-bottom: 7px;
     margin-left: 2px;
   }
+
+  &__wrapper {
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-datepicker-input.scss:11]: Fix formatting for this file

File: addon/templates/components/uni-datepicker-input.hbs [hunk 2]
```diff
@@ -2,13 +2,17 @@
...
+<div class="uni-datepicker-input__wrapper">
 {{input
   class="uni-input uni-input--bordered"
   value=formattedDate
   placeholder=placeholder
   click=(action "toggleDatepicker")
   readonly=isReadOnly}}
-
+  <div class="uni-datepicker-input__wrapper__icon">
+  {{inline-svg 'icons/calendar'}}
```
Comment (AndreJoaquim) [inline addon/templates/components/uni-datepicker-input.hbs]: Fix indentation

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

File: addon/styles/partials/_uni-datepicker-input.scss [hunk 3]
```diff
@@ -7,4 +7,32 @@
...
+      right: 20px;
+      top: 50%;
+      margin-top: -11px;
+      pointer-events: none;
+
+      svg {
+        width: 22px;
+
+        path{
+          fill: #00adef !important;
```
Comment (mmelo) [inline addon/styles/partials/_uni-datepicker-input.scss]: `fill: color("primary");`

File: public/assets/images/icons/calendar.svg [hunk 4]
```diff
@@ -1,6 +1 @@
...
-    </g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 17 18.59"><g fill="#575757"><path d="M16.43 1.38h-2.19V.57a.57.57 0 0 0-.57-.57H11.6a.57.57 0 0 0-.6.57v.81H6V.57A.57.57 0 0 0 5.4 0H3.33a.57.57 0 0 0-.57.57v.81H.57A.57.57 0 0 0 0 2v16a.57.57 0 0 0 .57.57h15.86A.57.57 0 0 0 17 18V2a.57.57 0 0 0-.57-.62zm-4.25-.23h.92v1.61h-.92zm-8.27 0h.92v1.61H3.9zM2.76 2.52v.81a.57.57 0 0 0 .57.57H5.4a.57.57 0 0 0 .6-.57v-.81h5v.81a.57.57 0 0 0 .57.57h2.07a.57.57 0 0 0 .57-.57v-.81h1.61v2.3H1.14v-2.3zM1.14 17.44V6h14.72v11.44z"/><path d="M14.26 9.52a.57.57 0 0 0 0-1.15h-2.09v-.8a.57.57 0 1 0-1.14 0v.81h-2.3v-.81a.57.57 0 1 0-1.14 0v.81H5.28v-.81a.57.57 0 1 0-1.14 0v.81h-1.4a.57.57 0 0 0 0 1.15h1.4v1.61h-1.4a.57.57 0 0 0 0 1.15h1.4v1.61h-1.4a.57.57 0 0 0 0 1.15h1.4v.81a.57.57 0 1 0 1.14 0V15h2.3v.81a.57.57 0 1 0 1.14 0V15H11v.81a.57.57 0 1 0 1.14 0V15h2.08a.57.57 0 0 0 0-1.15h-2.05v-1.57h2.08a.57.57 0 0 0 0-1.15h-2.08V9.52zm-9 0h2.3v1.61H5.28zm0 4.37v-1.61h2.3v1.61zm5.75 0H8.73v-1.61H11zm0-2.76H8.73V9.52H11z"/></g></svg>
```
Comment (mmelo) [inline public/assets/images/icons/calendar.svg]: newline EOF

**Review by mmelo — CHANGES_REQUESTED** [review]
minor changes requested

File: addon/styles/partials/_uni-datepicker-input.scss [hunk 5]
```diff
@@ -7,4 +7,32 @@
...
+  &__wrapper {
+    position: relative;
+    width: auto;
+
+    input{
+      width: 100%;
+      margin-right: 0;
+    }
+
+    &__icon{
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-datepicker-input.scss]: Missing space

File: addon/styles/partials/_uni-datepicker-input.scss [hunk 6]
```diff
@@ -7,4 +7,32 @@
...
+      position: absolute;
+      right: 20px;
+      top: 50%;
+      margin-top: -11px;
+      pointer-events: none;
+
+      svg {
+        width: 22px;
+
+        path{
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-datepicker-input.scss]: Missing space

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

**Comment by AndreJoaquim** [comment]
Please provide some screenshots

**Review by AndreJoaquim — APPROVED** [review]

**Review by mmelo — APPROVED** [review]

**Pull request merged**